Makefile

```
@@ -1,136 +1,105 @@
+		/root/.m2/repository/org/datasyslab/geotools-wrapper/$(GEOTOOLS_VERSION)/geotools-wrapper-$(GEOTOOLS_VERSION).jar \
+		./jars/
+	$(DKC) stop maven
+
+clean-jars:  ## Remove downloaded JARs
+	rm -rf ./jars/*.jar
+
+# ---------------------------------------------------------------------------
+# Census data (via siege_utilities inside the Python container)
+# ---------------------------------------------------------------------------
+
+.PHONY: download-census load-census
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)